mod/display.php

```diff
@@ -244,11 +238,11 @@ function display_content(App $a, $update = false, $update_uid = 0)
 	}
 
 	// We are displaying an "alternate" link if that post was public. See issue 2864
-	$is_public = Post::exists(['id' => $item_id, 'private' => [Item::PUBLIC, Item::UNLISTED]]);
+	$is_public = Post::exists(['uri-id' => $uri_id, 'private' => [Item::PUBLIC, Item::UNLISTED]]);
```

Comment:
Is this really equivalent? I was under the impression that `id = uri-id + uid` for unicity purposes?

Reply:
This is just a check if the post is public. The "public" value is identical for all entries.